main.cpp

```diff
@@ -63,85 +63,85 @@ dog.sleep();
 //  2)
 //  Noun: flashLight (i.e. pocket light)
 //  action 1: emit steady light
-emitContintuousLight();
+flashLight.emitContintuousLight();
 //  action 2: emit pulsating light
-blink();
+flashLight.blink();
 //  action 3: recharge from power supply
-recharge();
+flashLight.recharge();
 
 //  3)
 //  Noun: television
 //  action 1: show selected programm
-showProgram();
+television.showProgram();
 //  action 2: switch to a different program
-switchProgram();
+television.switchProgram();
 //  action 3: show on-screen menu
-showMenu();
+television.showMenu();
 
 //  4) 
 //  Noun: horse
 //  action 1: the horse walks
-walk();
+horse.walk();
 //  action 2: the horse trots
-trot();
+horse.trot();
 //  action 3: the horse gallops
-gallop();
+horse.gallop();
 
 //  5) swimmer (i.e. swimming athlete)
 //  Noun:
 //  action 1: the swimmer swims
-swim();
+swimmer.swim();
 //  action 2: the swimmer turns at the end of the lane
-turn();
+swimmer.turn();
 //  action 3: the swimmer finishes at the end of the lane
-finish();
+swimmer.finish();
 
 //  6)
 //  Noun: submarine
 //  action 1: the submarine emerges from the water
-emerge();
+submarine.emerge();
 //  action 2: the submarine submerges into the water
-submerge();
+submarine.submerge();
 //  action 3: the submarine raises its periscope
-raiseTelescope();
+submarine.raiseTelescope();
 
 //  7)
 //  Noun: balloon (i.e. a hot-air balloon)
 //  action 1: the open blast valve of the balloon's burner unit
-openBlastValveOfBurner();
+balloon.openBlastValveOfBurner();
 //  action 2: heat up the air by burning gas
-heatAirByBurningGas();
+balloon.heatAirByBurningGas();
 //  action 3: the balloon releases hot air through its vent
-releaseHotAirThroughVent();
+balloon.releaseHotAirThroughVent();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.